src/System.Collections.NonGeneric/src/System/Collections/ArrayList.cs

```diff
@@ -55,6 +57,7 @@ internal ArrayList(bool trash)
         // Constructs a ArrayList. The list is initially empty and has a capacity
         // of zero. Upon adding the first element to the list the capacity is
         // increased to _defaultCapacity, and then increased in multiples of two as required.
+        /// <summary>Initializes a new instance of the <see cref="T:System.Collections.ArrayList" /> class that is empty and has the default initial capacity.</summary>
```

Comment:
We also need to decide whether we want to keep existing /// comments or replace them with the content coming from IntelliSense. Currently, the tool doesn't replace the /// comments, if they already exist.